data/json/requirements/vehicle.json

```diff
@@ -1,8 +1,20 @@
 [
   {
-    "id": "vehicle_bolt",
+    "id": "vehicle_fasten",
```

Comment:
This change is breaking Aftershock which will break tests for everyone.  You need to also make this change in Aftershock items that used `vehicle_bolt` as well as any in Magiclysm if there are any.

Reply:
Right, I'll hop on that

Reply:
Alrighty then, everything seems to work. Thanks for reminding me about aftershock and magi